reporter/resources/buildings-points/transform.sql

```diff
@@ -0,0 +1,78 @@
+ALTER TABLE planet_osm_polygon ADD COLUMN "type" VARCHAR(255) NULL;
```

Comment:
Only transform.sql ?
There are a lot of files like this one : default.style, licence, ...

But transform.sql should disapear with https://github.com/AIFDR/inasafe/issues/1664
Should I fix this ?